events/wc_pandaria_events.txt

```diff
@@ -78,22 +76,799 @@ narrative_event = {
 				capital_scope = { region = world_pandaria }
 			}
 		}
-		name = EVTOPTA_WCPAN_100_expeditions
+		name = EVTOPTA_WCPND_100_expeditions
+	}
+}
+
+### ANKOAN COURTIERS ###
+
+# Ankoan diplomat appears at court
+character_event = {
+	id = WCPND.200
```

Comment:
All these events can be merged into 2 events (one for an ankoan, another for a pandaren) since they share the same triggers. When it's fired, the game randomly chooses the type of a courtier in the immediate block.